test/SourceKit/CodeFormat/indent-implicit-getter.swift

```diff
@@ -63,7 +63,7 @@ class var foo: Int {
 // CHECK: key.sourcetext: "    subscript(a: Int) -> Int {"
 // CHECK: key.sourcetext: "        let x = 3"
 // CHECK: key.sourcetext: "        return x"
-// CHECK: key.sourcetext: "    }"
+// CHECK: key.sourcetext: "        }"
```

Comment:
Hm, this looks a little suspicious. Should this have added indentation, @rintaro?

Reply:
\cc @nkcsgexi, this is a regression for indentation, please don't change the test.

Reply:
Thank you both. I found my mistake, fixed it, and changed the tests back. Hope it's good now.